wgengine/packet/icmp.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package packet
+
+type ICMPType uint8
+
+const (
+	ICMPEchoReply    = 0x00
```

Comment:
did you mean to make these typed constants? they're untyped right now.

Reply:
Yep, sorry, missed that.